src/selectors/sessionSelector.js

```diff
@@ -1,6 +1,6 @@
 import { createSelector } from 'reselect';
 
 export const getUser = createSelector(
```

Comment:
This selector is useless now :(

The goal of the selectors is to avoid recomputing data unless is necessary. In this case, we were avoiding to compute the function `.toJS()` that is expensive. But now, we are just accessing an object's property. It's more expensive the selector itself than the function to get the data.

Reply:
We could keep them around to calculate derived data, what do you think?
Or maybe it would be best to leave that to each project.